File: app/src/debug/java/com/antonio/samir/meteoritelandingsspots/ApplicationDebug.java
package com.antonio.samir.meteoritelandingsspots;

import com.facebook.stetho.Stetho;

public class ApplicationDebug extends com.antonio.samir.meteoritelandingsspots.Application {

    public void onCreate() {
        super.onCreate();

        //chrome://inspect
        Stetho.initializeWithDefaults(this);

    }

}

File: app/src/debug/java/com/antonio/samir/meteoritelandingsspots/ApplicationDebug.kt
package com.antonio.samir.meteoritelandingsspots

import com.facebook.stetho.Stetho
import kotlinx.coroutines.ExperimentalCoroutinesApi

@ExperimentalCoroutinesApi
class ApplicationDebug : Application() {

    override fun onCreate() {
        super.onCreate()

        //chrome://inspect
        Stetho.initializeWithDefaults(this)

    }

}
